src/components/instrument/instruments_list.ts

```diff
@@ -131,4 +131,11 @@ const instrumentsList:{[key:number]:string} = {
 
 export function getInstrumentByNumber(number: number):string  {
     return instrumentsList[number + 1].toLowerCase().replace(/[^a-z0-9 ]/g, '').replace(/ /g, '_')
+}
+
+export function getInstrumentsName(numbers: number[]) {
+    const list = numbers.map(e => {
+        return getInstrumentByNumber(e);
+    });
```

Comment:
This part can be simplified like this:
```suggestion
    const list = numbers.map(getInstrumentByNumber);
```